xline/src/server/xline_server.rs

```diff
@@ -31,23 +33,14 @@ type CurpServer = Rpc<Command>;
 #[allow(dead_code)] // Remove this after feature is completed
 #[derive(Debug)]
 pub struct XlineServer {
-    /// Server name
-    name: String,
-    /// Address of peers
-    peers: Vec<SocketAddr>,
+    /// State of current node
+    state: Arc<RwLock<State>>,
```

Comment:
why is it protected in a rwlock? I can't find a place to write to it. All references are read-only accesses.

Reply:
when I can get new leader_id from curp, the state will be updated, but in this PR it is read-only

Reply:
OK, I see.